lib/collection/src/collection_manager/optimizers/segment_optimizer.rs

```diff
@@ -128,11 +130,11 @@ pub trait SegmentOptimizer {
                 }
             };
             let locked_segment = segment.read();
-
             for vector_name in locked_segment.vector_names() {
                 let vector_size = locked_segment.available_vectors_size_in_bytes(&vector_name)?;
                 let size = bytes_count_by_vector_name.entry(vector_name).or_insert(0);
                 *size += vector_size;
+                total_payload_size_bytes += vector_size
```

Comment:
I don't think it is a correct way to estimate payload.

I think what we need to do is to simply check what's the current disk usage of the segment is, not try to estimate it with some form of proxy value.